test/uk/gov/hmrc/pushpullnotificationsgateway/services/CallbackValidatorSpec.scala

```diff
@@ -16,21 +16,19 @@
 
 package uk.gov.hmrc.pushpullnotificationsgateway.services
 
-import java.util.UUID.randomUUID
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future.{failed, successful}
-
 import org.mockito.{ArgumentMatchersSugar, MockitoSugar}
-import org.scalatest.{Matchers, WordSpec}
-
 import play.api.http.Status.INTERNAL_SERVER_ERROR
 import play.api.test.Helpers._
 import uk.gov.hmrc.http.{BadRequestException, JsValidationException, UpstreamErrorResponse}
-
 import uk.gov.hmrc.pushpullnotificationsgateway.connectors.OutboundProxyConnector
 import uk.gov.hmrc.pushpullnotificationsgateway.models.{CallbackValidation, CallbackValidationResult}
+import util.HmrcSpec
+
+import java.util.UUID.randomUUID
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future.{failed, successful}
 
-class CallbackValidatorSpec extends WordSpec with Matchers with MockitoSugar with ArgumentMatchersSugar {
+class CallbackValidatorSpec extends HmrcSpec with MockitoSugar with ArgumentMatchersSugar {
```

Comment:
MS and AMS are not required (HMRCSpec includes them). Appreciate this is not your code.

Reply:
Done